hotel-bookings-api/FrontEnd/Hotel/src/Bookings.js

```diff
@@ -0,0 +1,158 @@
+import { useState } from "react";
+
+function NewBookings({ props }) {
+  const [title, setTitle] = useState("");
+  const [firstName, setFirstName] = useState("");
+  const [surname, setSurname] = useState("");
+  const [email, setEmail] = useState("");
+  const [roomId, setRoomId] = useState("");
+  const [checkInDate, setCheckInDate] = useState("");
+  const [checkOutDate, setCheckOutDate] = useState("");
+
+  const handletitleChange = (event) => {
+    setTitle(event.target.value);
+  };
+
+  const handleFirstNameChange = (event) => {
+    setFirstName(event.target.value);
+  };
+  const handlesurNameChange = (event) => {
+    setSurname(event.target.value);
+  };
+
+  const handleEmailChange = (event) => {
+    setEmail(event.target.value);
+  };
+  const handleRoomIdChange = (event) => {
+    setRoomId(event.target.value);
+  };
+
+  const handlecheckInDateChange = (event) => {
+    setCheckInDate(event.target.value);
+  };
+  const handlecheckOutDateChange = (event) => {
+    setCheckOutDate(event.target.value);
+  };
+
+  const newMessage = {
+    title,
+    firstName,
+    surname,
+    email,
+    roomId,
+    checkInDate,
+    checkOutDate,
+  };
+
+  fetch("https://hotel-bookings-server.onrender.com/bookings", {
```

Comment:
Can you tell when and how many times the Post bookings api have  been called?

does you use case need a function handler to fetch the Post bookings api?

```
const onSubmitNewMessage = () => {
  fetch("https://hotel-bookings-server.onrender.com/bookings", {
  // your payload}.then(// your handle for the response
  ).then(
   // handle the response
  )
}
```